integration-tests/goss/goss-shared.yaml

```diff
@@ -180,6 +180,14 @@ interface:
       contain-element:
         have-prefix: '172.17'
 http:
+  https://httpbin.org/headers:
```

Comment:
this one will be overwrote with the one below on line 191 as its the same url.

Reply:
good catch 👍

Reply:
@ripienaar 

I wonder if that would also apply to the second http basic auth check (https://github.com/aelsabbahy/goss/blob/f95fac2bb3dab16dd932d000ebe09453beb767ff/integration-tests/goss/goss-shared.yaml#L203)? If so, I could update that while I am at it.

Reply:
Yeah I saw that too - it would yes